tests/e2e/Services/Databases/DatabasesBase.php

```diff
@@ -297,6 +297,174 @@ public function testCreateAttributes(array $data): array
         return $data;
     }
 
+    /**
+     * @depends testCreateDatabase
+     */
+    public function testNumericOnlyAttributes(array $data): void
+    {
+        $databaseId = $data['databaseId'];
+
+        $numeric_key_collection = $this->client->call(Client::METHOD_POST, '/databases/' . $databaseId . '/collections', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]), [
+            'collectionId' => ID::unique(),
+            'name' => 'Numeric Keys',
+            'documentSecurity' => true,
+            'permissions' => [
+                Permission::create(Role::user($this->getUser()['$id'])),
+            ],
+        ]);
+
+        $this->assertEquals(201, $numeric_key_collection['headers']['status-code']);
+        $this->assertEquals($numeric_key_collection['body']['name'], 'Numeric Keys');
+
+        $numericKeyId = $numeric_key_collection['body']['$id'];
+
+        $num_one = $this->client->call(Client::METHOD_POST, '/databases/' . $databaseId . '/collections/' . $numericKeyId . '/attributes/string', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]), [
+            'key' => '1',
+            'size' => 256,
+            'required' => false,
+            'array' => false,
+        ]);
+
+        $num_two_hundred = $this->client->call(Client::METHOD_POST, '/databases/' . $databaseId . '/collections/' . $numericKeyId . '/attributes/string', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]), [
+            'key' => '200',
+            'size' => 256,
+            'required' => false,
+            'array' => false,
+        ]);
+
+        $num_million = $this->client->call(Client::METHOD_POST, '/databases/' . $databaseId . '/collections/' . $numericKeyId . '/attributes/string', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]), [
+            'key' => '1000000',
+            'size' => 256,
+            'required' => false,
+            'array' => false,
+        ]);
```

Comment:
One attribute should be enough to test the fix, let's remove these other two to reduce the test time